src/tokens/transformation/tailwind/formatCssVars.js

```diff
@@ -11,9 +11,16 @@ module.exports = ({ dictionary, options, file }) => {
   const { outputReferences } = opts
   const groupedTokens = {
     // if you export the prefixes use token.path[0] instead of [1]
-    light: filteredTokens(dictionary, (token) => matchLightThemeToken(token)),
-    dark: filteredTokens(dictionary, (token) => matchDarkThemeToken(token)),
-    rest: filteredTokens(dictionary, (token) => token.type === 'color')
+    light: filteredTokens(dictionary, (token) =>
+      matchLightThemeToken(token, ['color'])
+    ),
+    dark: filteredTokens(dictionary, (token) =>
+      matchDarkThemeToken(token, ['color'])
+    ),
+    rest: filteredTokens(
+      dictionary,
+      (token) => !matchDarkThemeToken(token) && !matchLightThemeToken(token)
```

Comment:
What happens to the `effect.elevation` tokens? I think they're filtered out of `rest`, `light` & `dark`

Reply:
Yes, but that highlighted a problem which should be resolved in the latest commits.